ui/src/EventForm.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from "react";
+import { formatDate } from "./lib";
+import s from "./App.module.css";
+
+interface FormProps {
+  initialStartDate: string;
+  initialEndDate: string;
+  initialStartTime: string;
+  initialEndTime: string;
```

Comment:
Spoke with @elliedori about the possibility of having 2 props, start and end UTC, rather than these four, and this EventForm component handling the date string manipulations. Non-blocking, but something we discussed and considered.

Reply:
Ok, we will address it later as needed.